mdstat.go

```diff
@@ -95,6 +97,13 @@ func parseMDStat(mdStatData []byte) ([]MDStat, error) {
 		mdName := deviceFields[0] // mdx
 		state := deviceFields[2]  // active or inactive
 
+		mdType := "unknown"
+		if len(deviceFields) > 3 && isRaidType(deviceFields[3]) {
```

Comment:
If you move this to after `if len(lines) <= i+3 {` you can reduce the word checking to just the check for `(...)`.

Reply:
Actually, maybe this whole piece should be moved to the `evalStatusLine()` function.